e2e/cypress/test_module/cypress/support/step_definitions/orders-filters.steps.ts

```diff
@@ -0,0 +1,72 @@
+/// <reference types="cypress" />
+
+import { Given, When, Then } from "@badeball/cypress-cucumber-preprocessor";
+import {
+  deliveryMethods,
+  orderDeliveryStatuses
+} from "@cypress-e2e/fixtures/constants";
+
+When("I can see orders filter button", () => {
+  cy.getById("filter-button").should("be.visible");
+});
+
+When("I click on the orders filter button", () => {
+  cy.getById("filter-button").click();
+});
+
+When("I should see the orders filter drawer", () => {
+  cy.getById("orders-filter-drawer").should("be.visible");
+});
+
+When("I click on the apply button", () => {
+  cy.getById("apply-filters-button").click();
+});
+
+Then("I should see the orders filter drawer closed", () => {
+  cy.getById("orders-filter-drawer").should("not.be.visible");
+});
+
+Given(
+  "I select order status {string}",
+  (orderStatus: keyof typeof orderDeliveryStatuses) => {
```

Comment:
Can we create global type for it within cypress subproject?

Reply:
I just gave for it type `string` and it's okay. Thank you